### PR TITLE
Design suggestions

### PR DESCRIPTION
**Pull request by Jay-o-Way**
few suggestions, as also commented in  #1531
feel free to adapt where needed

### CODE REVIEW

File: WinUIGallery/Controls/DesignGuidance/TypographyControl.xaml [hunk 1]
```diff
@@ -99,50 +98,46 @@
             </Setter>
         </Style>
     </UserControl.Resources>
-    <Grid MinHeight="68" Background="Transparent">
```
Comment (niels9001) [inline WinUIGallery/Controls/DesignGuidance/TypographyControl.xaml:102]: I think we need the padding. 

Left (your changes) vs. right (current):

image

File: WinUIGallery/ControlPages/DesignGuidance/SpacingPage.xaml [hunk 2]
```diff
@@ -38,7 +38,16 @@
                 <Style.Setters>
                     <Setter Property="HorizontalAlignment" Value="Stretch" />
                     <Setter Property="MinHeight" Value="68" />
-                    <Setter Property="CornerRadius" Value="{StaticResource OverlayCornerRadius}" />
+                    <Setter Property="CornerRadius" Value="{StaticResource ControlCornerRadius}" />
+                </Style.Setters>
+            </Style>
+
+            <Style
+                x:Key="GridRowLightStyle"
```
Comment (niels9001) [inline WinUIGallery/ControlPages/DesignGuidance/SpacingPage.xaml:46]: Ah yes, this is good to do.

Should we rename these so they are consistent?

"GridRowCardStyle" and "GridRowLightCardStyle" or something like that?

And should we move these to App.xaml.cs so they can be re-used across these 2 pages?

**Review by niels9001 — CHANGES_REQUESTED** [review]
Thanks @jay-o-way, great changes! Some small things to fix, and then I'll merge this in!

File: WinUIGallery/ControlPages/DesignGuidance/SpacingPage.xaml (hunk 2 above)
Comment (Jay-o-Way) [reply WinUIGallery/ControlPages/DesignGuidance/SpacingPage.xaml:46]: Sure, do your thing! 👍🏻

**Pull request merged**